src/lib/libwasmfs.js

```diff
@@ -111,7 +111,11 @@ FS.init();
         if (!wasmFSPreloadingFlushed) {
           wasmFSPreloadedDirs.push({parentPath: parent, childName: part});
         } else {
-          FS.mkdir(current);
+          try {
+            FS.mkdir(current);
+          } catch (e) {
+            if (e.errno != {{{ cDefs.EEXIST }}}) throw e;
+          }
```

Comment:
Is there some test that can be updated now?   Maybe some test we couldn't run before but now we can?